test/realtime/ghost_test.exs

```diff
@@ -204,7 +260,8 @@ defmodule Realtime.GhostTest do
                  timepoint_id: "t2",
                  fraction_until_timepoint: 0.0
                },
-               route_status: :laying_over
+               route_status: :laying_over,
+               block_waivers: %{}
```

Comment:
I don't think you need this if you put `block_waivers: %{}` in the ghost's `defstruct`. (Same for vehicle and `vehicle_test.exs`.)

Reply:
Yep, good call. Fixed in a bigger commit that I'll get in here soon.